components/webui/imports/api/search/server/publications.js

```diff
@@ -19,6 +19,12 @@ import {searchJobCollectionsManager} from "./collections";
  */
 const COLLECTION_POLL_INTERVAL_MILLIS = 250;
 
+/**
+ * The maximum value that can be used as a polling interval in JavaScript.
+ * Reference: https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#maximum_delay_value
+ */
+const JS_MAX_DELAY_VALUE = 2147483647;
```

Comment:
```suggestion
const JS_MAX_DELAY_VALUE = Math.pow(2, 31) - 1;
```

Reply:
Lint shows there are magic number issues with this fix. How about using this number and note it is actually 2^31 - 1 in the comments?

Reply:
We can disable that lint rule for this line. I've added a suggestion.